packages/suite-web/scripts/s3sync.sh

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p awscli
+#! nix-shell -i bash -p awscli2
```

Comment:
What about [`packages/suite-web-landing/scripts/s3sync.sh`](https://github.com/trezor/trezor-suite/blob/a3c8b6190df74ede26f8a59a51c32be921dd99ad/packages/suite-web-landing/scripts/s3sync.sh)? shouldn't it also be updated?

Reply:
not needed for this fix but you're right, we should update it also.